lib/screens/home_screen.dart

```diff
@@ -1,39 +1,195 @@
+    });
+  }
+
+  void handleBrightnessChange() {
+    setState(() {
+      useLightMode = !useLightMode;
+      themeData = updateThemes(useLightMode);
+    });
+  }
+
+  void handleMaterialVersionChange() {
+    setState(() {
+      themeData = updateThemes(useLightMode);
+    });
+  }
+
+  void handleColorSelect(int value) {
+    setState(() {
+      themeData = updateThemes(useLightMode);
+    });
+  }
+
+  Widget createScreenFor(int screenIndex, bool showNavBarExample) {
+    switch (screenIndex) {
+      case 0:
+        return const AboutUsScreen();
+      case 1:
+        return const ProjectsScreen();
+      case 2:
+        return const EventsScreen();
+      case 3:
+        return const BlogScreen();
+      case 4:
+        return const ContactUsScreen();
+      default:
+        return const AboutUsScreen();
+    }
+  }
+
+  PreferredSizeWidget createAppBar() {
+    return AppBar(
+      title: const Text("Galactic Coders"),
+      actions: [
+        IconButton(
+          icon: useLightMode
+              ? const Icon(Icons.wb_sunny_outlined)
+              : const Icon(Icons.wb_sunny),
+          onPressed: handleBrightnessChange,
+          tooltip: "Toggle brightness",
+        ),
+        PopupMenuButton(
+          icon: const Icon(Icons.more_vert),
+          shape:
+              RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
+          itemBuilder: (context) {
+            return List.generate(colorOptions.length, (index) {
+              return PopupMenuItem(
+                  value: index,
+                  child: Wrap(
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.only(left: 10),
+                        child: Icon(
+                          index == true
+                              ? Icons.color_lens
+                              : Icons.color_lens_outlined,
+                          color: colorOptions[index],
+                        ),
+                      ),
+                      Padding(
+                          padding: const EdgeInsets.only(left: 20),
+                          child: Text(colorText[index]))
+                    ],
+                  ));
+            });
+          },
+          onSelected: handleColorSelect,
+        ),
+      ],
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(title: const Text('GalacticCoders')),
-      body: Container(
-        decoration: const BoxDecoration(
-            image: DecorationImage(
-          image: AssetImage('assets/spiral-galaxy-synthwave.png'),
-          fit: BoxFit.cover,
-        )),
-        child: Center(
-          child: Container(
-            padding: const EdgeInsets.all(24),
-            decoration: const BoxDecoration(
-              borderRadius: BorderRadius.all(Radius.circular(20)),
-              color: Colors.white70,
+    return MaterialApp(
```

Comment:
Is this a test to try out different material themes or so?

There's, NORMALLY, really only ever one MaterialApp-widget ever needed and it should be at the top of your application. So depending on the idea is here I'd probably remove the MaterialApp widget and put the scaffold back.